local-cluster/src/cluster_tests.rs

```diff
@@ -91,7 +92,9 @@ pub fn verify_balances<S: ::std::hash::BuildHasher>(
     expected_balances: HashMap<Pubkey, u64, S>,
     node: &ContactInfo,
 ) {
-    let client = create_client(node.client_facing_addr());
+
+    let client = create_client(node.client_facing_addr().0, node.client_facing_addr().1);
+
```

Comment:
prefer calling `client_facing_addr()` once and destructure into temporaries (again, throughout)
```suggestion
    let (rpc, tpu) = node.client_facing_addr();
    let client = create_client(rpc, tpu);
```

Reply:
changed, thank you!